admin-dev/themes/default/template/header.tpl

```diff
@@ -262,9 +262,7 @@
              data-html="true"
              title="{$smarty.capture.title|htmlspecialchars}"
           >
-            <i class="material-icons"
-               style="{if isset($maintenance_allow_admins)}color: #72c279;{/if}"
```

Comment:
As I mentioned in the pre-review, this should be highlighted one way or another. It should stay because it's a feature added recently that allows admins to access the store even if the maintenance is enabled and their IP hasn't been added to the allow list.

Reply:
I have highlighted the maintenance link in the header in CSS after a talk with designers ✅
However, I think we have an issue because the maintenance link in the header seems to never be displayed (with or without the rebranding).

Reply:
Yes, you're right, but according to designers we will use only one color for this.